homeassistant/components/sun.py

```diff
@@ -30,6 +30,10 @@
 
 STATE_ATTR_AZIMUTH = 'azimuth'
 STATE_ATTR_ELEVATION = 'elevation'
+STATE_ATTR_NEXT_DAWN = 'next_dawn'
+STATE_ATTR_NEXT_DUSK = 'next_dusk'
+STATE_ATTR_NEXT_MIDNIGHT = 'next_midnight'
```

Comment:
I think they're useful (mostly next_noon and next_dusk in my case). Already have external command line sensors for them but having this in the sun component would simplify the setup.